metadrive/type.py

```diff
@@ -125,31 +141,30 @@ def is_traffic_light_in_green(cls, light):
     def is_traffic_light_in_red(cls, light):
         return light in [cls.LANE_STATE_STOP, cls.LANE_STATE_ARROW_STOP, cls.LANE_STATE_FLASHING_STOP]
 
-
-class TrafficLightStatus:
-    GREEN = 1
-    RED = 2
-    YELLOW = 3
-    UNKNOWN = 4
-
     @classmethod
-    def semantics(self, status):
-        if status == self.GREEN:
-            return "Traffic Light: Green"
-        if status == self.RED:
-            return "Traffic Light: Red"
-        if status == self.YELLOW:
-            return "Traffic Light: Yellow"
-        if status == self.UNKNOWN:
-            return "Traffic Light: Unknown"
+    def parse_light_status(cls, status: int, simplifying=True):
+        """
+        Parse light status from ENUM to STR
+        """
+        ret = cls.LIGHT_ENUM_TO_STR[status]
+        if simplifying:
+            return cls.simplify_light_status(ret)
+        else:
+            return ret
 
     @classmethod
-    def color(self, status):
-        if status == self.GREEN:
-            return [0, 255, 0]
-        if status == self.RED:
-            return [1, 255, 0]
-        if status == self.YELLOW:
-            return [255, 255, 0]
-        if status == self.UNKNOWN:
-            return [180, 180, 180]
+    def simplify_light_status(cls, status: str):
+        """
+        Convert status to red/yellow/green/unknown
+        """
+        if status is cls.LANE_STATE_UNKNOWN or status is cls.LIGHT_UNKNOWN:
+            return cls.LIGHT_UNKNOWN
+        elif status in [cls.LANE_STATE_ARROW_STOP, cls.LANE_STATE_STOP, cls.LANE_STATE_FLASHING_STOP, cls.LIGHT_RED]:
```

Comment:
Flashing Stop should be counted into UNKNOWN.

Reply:
IDK, it has a ```stop``` in its name, and I can not find any explanation about this type on internet

Reply:
Flashing Red light = stop sign in US. Just set it to UNKNOWN.

Reply:
Done